implants/lib/eldritch/Cargo.toml

```diff
@@ -40,6 +40,8 @@ windows-sys = { workspace = true, features = [
     "Win32_Security",
 ]}
 whoami = { workspace = true }
+pnet = "0.34.0"
+ipnetwork = "0.20.0"
```

Comment:
Please add these to the workspace root and use workspace true in the package. This way if another package wants to use a crate they'll be kept in step with the main version. If specific libs need to break with the current workspace version we can do that but should avoid it.